src/WinXP/index.js

```diff
@@ -277,7 +284,13 @@ function WinXP() {
   }
   function onClickModalButton(text) {
     dispatch({ type: CANCEL_POWER_OFF });
-    dispatch({ type: ADD_APP, payload: appSettings.Error });
+    dispatch({
+      type: ADD_APP,
+      payload: {
+        ...appSettings.Error,
+        injectProps: { message: 'Something wrong!' },
```

Comment:
```suggestion
        injectProps: { message: 'Something's wrong!' },
```

Reply:
"Something wrong!" Doesn't sound right.

Reply:
That will break the single quotes. Escape needed or ``

Reply:
Fixed

Reply:
I meant backticks for the whole strings.